angular.json

```diff
@@ -23,18 +23,13 @@
                         "main": "apps/docs/src/main.ts",
                         "polyfills": "apps/docs/src/polyfills.ts",
                         "tsConfig": "apps/docs/tsconfig.app.json",
-                        "assets": [
-                            "apps/docs/src/favicon.ico",
-                            "apps/docs/src/assets"
-                        ],
+                        "assets": ["apps/docs/src/favicon.ico", "apps/docs/src/assets"],
```

Comment:
you can remove this formatting. As No changes is made in this file.

Reply:
removed formatting